materialize-bigquery/.snapshots/TestApply

```diff
@@ -373,6 +373,33 @@
 ]
 --- End remove new binding ---
 
+--- Begin replace original binding table ---
+* Table list: [first-table flow_checkpoints_v1 flow_materializations_v2 second-table]:
+* Schema for estuary-theatre.testing.first-table:
+[
+  {
+    "Name": "flow_document",
+    "Nullable": "NO",
+    "Type": "STRING"
+  },
+  {
+    "Name": "key",
+    "Nullable": "NO",
+    "Type": "STRING"
+  },
+  {
+    "Name": "optionalVal1",
+    "Nullable": "YES",
+    "Type": "INT64"
+  },
+  {
+    "Name": "requiredVal1",
+    "Nullable": "NO",
```

Comment:
In these snapshots, you can tell the table was replaced because it just has the original fields from the "base" spec, and that this `requiredVal1` is not nullable. The progression of the tests is to create the table from the "base" spec, add some columns and drop the nullability, and then re-create the table from the original "base" spec. So we can see that the table has conclusively been replaced.